docs/src/providers/features.md

```diff
@@ -0,0 +1,5 @@
+# Features
+
+### Fuel-core lib
+
+The `fuel-core-lib` feature allows us to run a fuel-core node without installing the fuel-core binary on the local machine. This entails downloading all the dependencies needed to run the fuel-core node.
```

Comment:
You should explain what dependencies it requires, maybe even put an example `Cargo.toml` file like in the `Setup` part.